docs/Compile.rst

```diff
@@ -200,39 +280,141 @@ On Windows, DFHack replaces the SDL library distributed with DF.
 
 Dependencies
 ------------
-You will need some sort of Windows port of git, or a GUI. Some examples:
+You will need the following:
+
+* Microsoft Visual Studio 2010 SP1, with the C++ language
+* Git
+* CMake
+* Perl with XML::LibXML and XML::LibXSLT
+
+  * It is recommended to install StrawberryPerl, which includes both.
+
+Microsoft Visual Studio 2010 SP1
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The free Express version is sufficient.
+
+You can grab it from `Microsoft's site <http://download.microsoft.com/download/1/E/5/1E5F1C0A-0D5B-426A-A603-1798B951DDAE/VS2010Express1.iso>`_.
+
+You'll also need the Visual Studio 2010 SP1 update, which is obtained from
+Windows Update. After installing Visual Studio, be sure to go to Windows Update
+and check for and install the SP1 update. If no update is found, check that
+your Windows Update settings include "Updates from all Microsoft products".
+
+You can confirm whether you have SP1 by opening the Visual Studio 2010 IDE
+and selecting About from the Help menu.  If you have SP1 it will have *SP1Rel*
+at the end of the version number, for example: *Version 10.0.40219.1 SP1Rel*
+
+It is vital that you use SP1 as, while building with the original release
+of Visual Studio 2010 (RTM) may succeed, it will result in non-working DFHack
+binaries that crash when connecting to Dwarf Fortress.
+
+Additional dependencies: installing with the Chocolatey Package Manager
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The remainder of dependencies - Git, CMake and StrawberryPerl - can be most
+easily installed using the Chocolatey Package Manger. Chocolatey is a
+\*nix-style package manager for Windows. Think "``apt-get`` for Windows."
+
+Chocolatey is the recommended way of installing the required dependencies
```

Comment:
No discussion has occurred, it's merely my own strong recommendation.

Chocolatey installs the exact same files as one would manually, just it sets up PATHs etc correctly, installs any needed dependencies, provides ongoing application management (upgrades etc), and is a useful system to have in place for any more complex requirements eg when I reference it under Docs as a one-line install for Python/Pip for Sphinx.

Chocolatey has been around 4.5 years and has 3000+ unique packages with a pretty active community, so it's pretty stable.  And we've had known issues in #dfhack with people not getting PATHs set up right, so known-good config is a strong benefit in my view.  So I'd have to say it's little different to recommending apt-get as the standard for Ubuntu. 

But I can tone down the nature of the recommendation if you think it's necessary.